info-v4-commands.go

```diff
@@ -240,16 +240,53 @@ func (adm *AdminClient) SetInfo(ctx context.Context, poolIndex int, setIndex int
 		return ExtendedErasureSetInfo{}, err
 	}
 
-	// Check response http status code
 	if resp.StatusCode != http.StatusOK {
 		return ExtendedErasureSetInfo{}, httpRespToErrorResponse(resp)
 	}
 
-	// Unmarshal the server's msgp response
 	var info ExtendedErasureSetInfo
 	if err = info.DecodeMsg(msgp.NewReader(resp.Body)); err != nil {
 		return ExtendedErasureSetInfo{}, err
 	}
 
 	return info, nil
 }
+
+// DriveInfoOpts ask for additional data from the server
+// this is not used at the moment, kept here for future
+// extensibility.
+//
+//msgp:ignore DriveInfoOpts
+type DriveInfoOpts struct{}
+
+// DriveInfo returns pool information about a specific pool referenced by poolIndex
+func (adm *AdminClient) DriveInfo(ctx context.Context, poolIndex, setIndex, diskIndex int, options ...func(*DriveInfoOpts)) (Disk, error) {
+	srvOpts := &DriveInfoOpts{}
+
+	for _, o := range options {
+		o(srvOpts)
+	}
+
+	values := make(url.Values)
+	resp, err := adm.executeMethod(ctx,
+		http.MethodGet,
+		requestData{
+			relPath:     adminAPIPrefixV4 + fmt.Sprintf("/drive/%d/%d/%d", poolIndex, setIndex, diskIndex),
```

Comment:
```suggestion
			relPath:     adminAPIPrefixV4 + fmt.Sprintf("/drive/%d/%d/%d", poolIndex, setIndex, driveIndex),
```

Reply:
moved the naming to Disk Info from the consumer perspective since we can't rename the Disk struct